src/components/StatsBoxGrid/useStatsBoxGrid.tsx

```diff
@@ -1,259 +1,100 @@
-    const localeForStatsBoxNumbers = getLocaleForNumberFormat(locale! as Lang)
-
-    const formatTotalStaked = (amount: number): string => {
-      return new Intl.NumberFormat(localeForStatsBoxNumbers, {
-        notation: "compact",
-        minimumSignificantDigits: 3,
-        maximumSignificantDigits: 4,
-      }).format(amount)
-    }
+  const [selectedRangeTxs, setSelectedRangeTxs] = useState<string>(RANGES[0])
 
-    const formatTVL = (tvl: number): string => {
-      return new Intl.NumberFormat(localeForStatsBoxNumbers, {
-        style: "currency",
-        currency: "USD",
-        notation: "compact",
-        minimumSignificantDigits: 3,
-        maximumSignificantDigits: 4,
-      }).format(tvl)
-    }
+  const localeForNumberFormat = getLocaleForNumberFormat(locale! as Lang)
 
-    const formatTxs = (txs: number): string => {
-      return new Intl.NumberFormat(localeForStatsBoxNumbers, {
-        notation: "compact",
-        minimumSignificantDigits: 3,
-        maximumSignificantDigits: 4,
-      }).format(txs)
-    }
-
-    const formatNodes = (nodes: number): string => {
-      return new Intl.NumberFormat(localeForStatsBoxNumbers, {
-        minimumSignificantDigits: 3,
-        maximumSignificantDigits: 4,
-      }).format(nodes)
-    }
-
-    const weiToRoundedEther = (wei: number): number => Math.floor(wei * 1e-18)
-
-    const fetchTotalStaked = async (): Promise<void> => {
-      const { href: ethstoreLatest } = new URL(
-        "api/v1/ethstore/latest",
-        "https://beaconcha.in"
-      )
-      try {
-        // 1- Use initial call to `latest` to fetch current Beacon Chain "day" (for use in secondary fetches)
-        const ethStoreResponse = await getData<IFetchEthstoreResponse>(
-          ethstoreLatest
-        )
-        const {
-          data: { day, effective_balances_sum_wei },
-        } = ethStoreResponse
-        const valueTotalEth = weiToRoundedEther(effective_balances_sum_wei)
-        const currentValueTotalEth = formatTotalStaked(valueTotalEth)
-        const MS_PER_DAY = 1000 * 60 * 60 * 24
-        const [DAYS_TO_FETCH, DAY_DELTA] = [90, 5]
-        const data = [
-          {
-            timestamp: new Date().getTime(),
-            value: valueTotalEth,
-          },
-        ]
-        // 2- Perform multiple API calls to fetch data for the last 90 days, `getData` for caching
-        for (let i = DAY_DELTA; i <= DAYS_TO_FETCH; i += DAY_DELTA) {
-          const lookupDay = day - i
-          const timestamp = new Date().getTime() - i * MS_PER_DAY
-          const { href: ethstoreDay } = new URL(
-            `api/v1/ethstore/${lookupDay}`,
-            "https://beaconcha.in"
-          )
-          const {
-            data: { effective_balances_sum_wei: sumWei },
-          } = await getData<IFetchEthstoreResponse>(ethstoreDay)
-          const value = weiToRoundedEther(sumWei)
-          data.push({ timestamp, value })
+  const totalEtherStaked =
+    "error" in totalEthStaked
```

Comment:
This PR left in the "error" state handling before we spoke more about it. Shouldn't hurt for now, but separately we can work to remove these as our data fetching becomes more robust on the server